campaignion_email_to_target/src/Tokens.php

```diff
@@ -14,24 +14,26 @@ abstract class Tokens {
    */
   public static function submissionTokens(array $tokens, Submission $submission) {
     $replacements = [];
-    if (!empty($tokens['email-to-target-messages']) || !empty($tokens['target-messages'])) {
-      $components = $submission->webform->componentsByType('e2t_selector');
-      $messages = [];
-      foreach ($components as $cid => $component) {
-        $data = $submission->valuesByCid($cid);
-        foreach ($data as $serialized) {
-          $mail = unserialize($serialized);
-          $message = new Message($mail['message']);
-          $messages[] = $message;
+
+    foreach (['email-to-target-messages', 'target-messages'] as $token) {
+      if (!empty($tokens[$token])) {
+        $components = $submission->webform->componentsByType('e2t_selector');
+        $messages = [];
+        foreach ($components as $cid => $component) {
+          $data = $submission->valuesByCid($cid);
+          foreach ($data as $serialized) {
+            $mail = unserialize($serialized);
+            $message = new Message($mail['message']);
+            $messages[] = $message;
+          }
         }
+        $themehook = 'campaignion_target_messages_token';
+        $text = theme([$themehook, $themehook . '_' . $submission->node->nid], [
+          'messages' => $messages,
+          'submission' => $submission,
+        ]);
+        $replacements[$tokens[$token]] = $text;
       }
-      $themehook = 'campaignion_target_messages_token';
-      $text = theme([$themehook, $themehook . '_' . $submission->node->nid], [
-        'messages' => $messages,
-        'submission' => $submission,
-      ]);
-      $replacements[$tokens['target-messages']] = $text;
-      $replacements[$tokens['email-to-target-messages']] = $text;
```

Comment:
Why was this change needed? I think in this case adapting the test would have been the better option. (If the previous option made a test fail.)